tests/js/panelsTests.js

```diff
@@ -25,6 +25,14 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The correct choice should be checked", selection, inputs.filter(":checked").val());
     };
 
+    gpii.tests.firstDiscovery.panel.utils.verifyTooltipContents = function (domElems, currentValue, idToContent, controlValues, stringArray, messages) {
+        fluid.each(domElems, function (elem, idx) {
+            var tooltipLabelSuffix = controlValues[idx] === currentValue ? "-tooltipAtSelect" : "-tooltip";
+            elem = $(elem);
+            jqUnit.assertEquals("The tooltip definition for the element #" + idx + " has been populated correctly", messages[stringArray[idx] + tooltipLabelSuffix], idToContent[elem.attr("id")]);
```

Comment:
If this is being used for both the inputs and labels, it might be hard to distinguish if one is working but not the other. Can we include more information about the element that is being tested to help identify it.